release/BASE_32_VERSION_MAPPING.md

```diff
@@ -0,0 +1,75 @@
+| CM              | 6.12.0           | 2021-08-10 |                                                                                                                                                                                                                       |
+| CN              | 6.13.0           | 2021-08-23 |                                                                                                                                                                                                                       |
+| CO              | 6.14.0           | 2021-09-13 |                                                                                                                                                                                                                       |
+| CP              | 6.15.0           | 2021-10-11 |                                                                                                                                                                                                                       |
+| CQ              | 6.15.1           | 2021-10-21 |                                                                                                                                                                                                                       |
+| CR              | 6.15.2           | 2021-11-01 |                                                                                                                                                                                                                       |
+| CS              | 6.15.3           | 2021-11-10 |                                                                                                                                                                                                                       |
+| CT              | 6.15.4           | 2021-11-25 |                                                                                                                                                                                                                       |
+| CU              | 6.15.5           | 2021-12-2  |                                                                                                                                                                                                                       |
+| CV              | 6.16.0           | 2021-12-15 |                                                                                                                                                                                                                       |
+| DA              | 6.17.0           | 2022-01-24 |                                                                                               
+| DB              | 7.0.0-alpha.2    | 2022-02-08                                                                                                     |
```

Comment:
Minor: There is one pipe missing at the end of this line and one before this line @zoeradkani

Reply:
fixed